composer.json

```diff
@@ -27,6 +27,9 @@
     "entrypoint": "Bolt\\Geolocation\\Extension",
     "screenshots": [
       "screenshots/backend-zoom.gif"
-    ]
+    ],
+    "branch-alias": {
+      "dev-master": "1.x-dev"
+    }
```

Comment:
why is this needed?

Reply:
My bad, for upstream not needed, removed.